idl/arm_batch.pro

```diff
@@ -62,6 +62,8 @@ pro arm_batch, temp_path, output_path
 	
 ; Get latest events from SSW database	    
     print, 'concating AR summary'
+    help, date_struct
```

Comment:
Are these meant to have been left here in the script? Perhaps you were checking them and forgot to remove..